### PR TITLE
Added EXAScaler driver

### PR DESCRIPTION
**Pull request by Qeas**

### CODE REVIEW

**Review by goldranjith — APPROVED** [review]
LGTM

File: book/src/drivers.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [DigitalOcean Block Storage](https://github.com/digitalocean/csi-digitalocean) | `dobs.csi.digitalocean.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for DigitalOcean Block Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion
 [DriveScale](https://github.com/DriveScale/k8s-plugins) | `csi.drivescale.com` | v1.0 |A Container Storage Interface (CSI) Driver for DriveScale software composable infrastructure solution | Persistent | Read/Write Single Pod | Yes | 
 [Ember CSI](https://ember-csi.io) | `[x].ember-csi.io` | v0.2, v0.3, v1.0 | Multi-vendor CSI plugin supporting over 80 Drivers to provide block and mount storage to Container Orchestration systems. | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot
+[DDN EXAScaler](https://github.com/DDNStorage/exa-csi-driver) | `exa.csi.ddn.com` | v1.0, v1.1 | A Container Storage Interface (CSI) Driver for DDN EXAScaler filesystems | Persistent | Read/Write Multiple Pods | Yes | Expansion
```
Comment (xing-yang) [inline book/src/drivers.md:41]: Can you move this line up to before "Dell" CSI drivers?

Comment (Qeas) [reply book/src/drivers.md]: Done.

**Pull request merged**